### DISCUSSION TITLE
DeclarativeNetRequest: matching based on response headers

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBOL-home#75] [General]
https://github.com/w3c/webextensions/issues/460

Chrome is working on blocking by response.

Maybe ask to support response size matching, which will work for no-large-media?

@gorhill ?

**Comment by gorhill** [comment]
I think the worklet proposal is more appropriate as a solution: https://github.com/w3c/webextensions/issues/461
